### PR TITLE
Add a duplicate counter to the video analyzer

### PR DESCRIPTION
**Pull request by alvestrand**
**Description**
Adds a duplicate count to the display of the video analyzer

**Purpose**
To see how many duplicates are generated, and when.

### CODE REVIEW

File: src/content/peerconnection/video-analyzer/js/main.js [hunk 1]
```diff
@@ -253,6 +258,14 @@ function videoAnalyzer(chunk, controller) {
     interFrameCount++;
     interFrameLastSize = chunk.data.byteLength;
   }
+  if (chunk.type === prevFrameType &&
+      chunk.timestamp === prevFrameTimestamp &&
+      chunk.synchronizationSource === prevFrameSynchronizationSource) {
+    duplicateCount += 1;
```
Comment (guidou) [inline src/content/peerconnection/video-analyzer/js/main.js]: duplicateCount++ ?

**Review by guidou — APPROVED** [review]
lgtm % nit

File: src/content/peerconnection/video-analyzer/js/main.js (hunk 1 above)
Comment (alvestrand) [reply src/content/peerconnection/video-analyzer/js/main.js]: Done!

**Comment by fippo** [comment]
hah, saw this one too late. I *think* the duplicates are not guaranteed to have a distance of 1. A better approach might be to (also?) look for a timestamp that is smaller than the last received one.

**Pull request merged**